### PR TITLE
Flow support

### PR DESCRIPTION
**Pull request by avaer**

### CODE REVIEW

File: blockchain.js [hunk 1]
```diff
@@ -0,0 +1,626 @@
...
+};
+const _bakeContract = async (contractKeys, contractSource) => {
+  const acctResponse = await flow.sdk.send(await flow.sdk.pipe(await flow.sdk.build([
+    flow.sdk.getAccount(contractKeys.address),
+  ]), [
+    flow.sdk.resolve([
+      flow.sdk.resolveParams,
+    ]),
+  ]), { node: flowConstants.host });
+  const seqNum = acctResponse.account.keys[0].sequenceNumber;
```
Comment (ChrisEddy) [inline blockchain.js:51]: may want to check for undefined along this object path

File: blockchain.js [hunk 2]
```diff
@@ -0,0 +1,626 @@
...
+  // set up ft
+  {
+    const acctResponse = await flow.sdk.send(await flow.sdk.pipe(await flow.sdk.build([
+      flow.sdk.getAccount(userKeys.address),
+    ]), [
+      flow.sdk.resolve([
+        flow.sdk.resolveParams,
+      ]),
+    ]), { node: flowConstants.host });
+    const seqNum = acctResponse.account.keys[0].sequenceNumber;
```
Comment (ChrisEddy) [inline blockchain.js:93]: This is duplicated 8 times in the code, maybe break this out as it's own function if possible.

File: blockchain.js (hunk 1 above)
Comment (avaer) [reply blockchain.js:51]: As far as I can tell the APIs always return this format.

File: blockchain.js (hunk 2 above)
Comment (avaer) [reply blockchain.js:93]: Yes, that's a good call

**Comment by ChrisEddy** [comment]
Just a couple review comments, file conflict needs resolve before merge.

**Pull request merged**